build

```diff
@@ -0,0 +1 @@
+build.sh
```

Comment:
I'd much rather remove this symlink entirely, only a few repositories are still using it currently.

Reply:
I'm fine with that.